identity-service/src/tracer.js

```diff
@@ -0,0 +1,54 @@
+const {
+  trace,
+  diag,
+  DiagConsoleLogger,
+  DiagLogLevel
+} = require('@opentelemetry/api')
+
+const { registerInstrumentations } = require('@opentelemetry/instrumentation')
+const { NodeTracerProvider } = require('@opentelemetry/sdk-trace-node')
+const { Resource } = require('@opentelemetry/resources')
+const {
+// SemanticAttributes,
+  SemanticResourceAttributes: ResourceAttributesSC
+} = require('@opentelemetry/semantic-conventions')
+const { HttpInstrumentation } = require('@opentelemetry/instrumentation-http')
+const { ExpressInstrumentation } = require('@opentelemetry/instrumentation-express')
+const { BunyanInstrumentation } = require('@opentelemetry/instrumentation-bunyan')
+
+// Not functionally required but gives some insight what happens behind the scenes
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.INFO)
```

Comment:
what does this do

Reply:
The only logs I've seen come out of this are warnings if someone tries to initialize tracing twice. This helped me move the call to `setupTracing()` from `app.js` to `index.ts`

Reply:
sure but like, literally what is DiagConsoleLogger? can't tell from code here

Reply:
Oh, it's a diagnostics logger for open telemetry. It's for debugging open telemetry issues which is kinda meta but that's why it's a separate logger